__tests__/app.test.js

```diff
@@ -0,0 +1,33 @@
+const db = require("../db/connection");
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const data =require("../db/data/test-data/index")
+
+afterAll(() => {
+  return db.end();
+});
+beforeEach(() => {
+  return seed(data);
+});
+
+describe("/api/topics", () => {
+  describe("GET", () => {
+    test("status: 200 which is OK", () => {
+      return request(app).get("/api/topics").expect(200);
+    });
+    test("status: 200 responds with array of topics objects with correct properties", () => {
+      return request(app)
+        .get("/api/topics")
+        .then(({ body }) => {
+          const { topics } = body;
+          expect(Array.isArray(topics)).toBe(true);
+          expect(topics.length).toBeGreaterThan(0);
+          topics.forEach((topic) => {
+            expect(typeof topic.description).toBe("string");
+            expect(typeof topic.slug).toBe("string");
+          });
+        });
+    });
+  });
```

Comment:
Few bits of feedback on this test:

- It's fine to just do this as 1 big integration test rather than 2. You can have 1 test which checks the status 200 and response all together rather than separate ones.
- Good working checking it's an array and that the length is above 0 so to ensure we don't end up with a 'false positive'